README.md

```diff
@@ -4,6 +4,12 @@
 
 Discord bot for Eddie Jaoude's Discord server
 
+---
+
+Feel free to [create an issue](https://github.com/EddieJaoudeCommunity/EddieBot/issues) or make a Pull Request 😃. Please see our [Contributing](CONTRIBUTING.md) file first, before making new commits or opening a PR. We appreciate it ❤️!
```

Comment:
having two dividers seems a bit strange since this is at the top of the readme. What do you think of making this the first header, like this:
```suggestion
# Contributing

Feel free to [create an issue](https://github.com/EddieJaoudeCommunity/EddieBot/issues) or make a Pull Request 😃. Please see our [Contributing](CONTRIBUTING.md) file first, before making new commits or opening a PR. We appreciate it ❤️!
```

Or perhaps move this section towards the end of the file (I think most repos have the contribution part at the end also).

What do you think @AllanRegush ?

Reply:
I think that looks good 👍